core/base-service/coalesce-badge.js

```diff
@@ -116,6 +117,19 @@ export default function coalesceBadge(
     const overrideLogoSvgBase64 = decodeDataUrlFromQueryParam(overrideLogo)
     if (overrideLogoSvgBase64) {
       logoSvgBase64 = overrideLogoSvgBase64
+      logoSize = coalesce(overrideLogoSize, serviceLogoSize)
+
+      if (logoSize === 'auto') {
+        const [, base64Image] = overrideLogoSvgBase64.split(';base64,')
+        const dimensions = base64Image
+          ? imageSize(Buffer.from(base64Image, 'base64'))
```

Comment:
Feels like there is a risk `imageSize` could throw here on spurious inputs and I don't want that to be an unhandled exception. There's no guarantee what the user has entered here is intelligible.

I think we should wrap the bit where we attempt to decode the base64 and parse the image in a try/catch and either

- re-throw any exception as an `InvalidParameter` or
- just ignore it and don't attempt to apply a custom `logoWidth` (but attempt to render the badge with whatever input we've been given)